src/main.rs

```diff
@@ -0,0 +1,182 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::{StreamExt};
+use  std::fs;
+use text_io::read;
+
+const TEST_DATABASE: &str = "codeforces-data-model";
```

Comment:
It's great that you've made this a constant -- makes the code a lot easier to read.